base/BUILD.gn

```diff
@@ -4210,11 +4210,6 @@ test("base_unittests") {
         "trace_event/memory_allocator_dump_unittest.cc",
         "trace_event/process_memory_dump_unittest.cc",
         "trace_event/typed_macros_unittest.cc",
-
-        # TODO: The below tests are key to ensuring tracing works.
-        # Enable them ASAP.
-        "test/trace_event_analyzer_unittest.cc",
-        "trace_event/trace_event_unittest.cc",
```

Comment:
If they are key, and this PR is about enabling tracing, do we want to get them work in this PR? What's blocking them?

Reply:
If you look above, this commit is actually adding them back to list of tests we run, I just temporarily disabled it while working on the change